File: C_programming/multable.c
//13.wap to accept a no. and print table of that no.

int main()
{
	int num;
	int i=0;
	 printf("program to generate multiplication table : ");
	 printf("\n\n enter no. to generate multipliction table : ");
	 scanf("%d",&num);
	 
	 printf("\n\n multiplication table of %d : \n",num);
	 printf("-------------------------------------\n\n");
	 
	 for(i;i<11;i++)
	 {
	 	printf("%d * %d =%ld \n\n",i,num, i*num);
	 }
	
}

File: Java_Programs/Java_Programs/MulTable.java
//9.wap to accept a no. and print table of that no.
import java.util.*;
class MulTable
{
public static void main(String args[])
{
   int num,result=0;
   int i=1;
   Scanner s=new Scanner(System.in);
   System.out.println("enter a number :");
   num=s.nextInt();
   System.out.println("table of "+num+" is :");
   while(i<=10)
   {
	   result=num*i;
	   System.out.println(num+ " * " +i+ " = " +result);

	   i=i+1;
   }
 }
}